discord/http.py

```diff
@@ -252,7 +252,9 @@ async def request(
                 headers["X-Audit-Log-Reason"] = _uriquote(reason, safe="/ ")
 
         kwargs["headers"] = headers
-
+        print(f"For Route: {route.url}")
+        print("Showing headers right before request:")
+        print(kwargs["headers"])
```

Comment:
Shouldn't these be removed? 
```suggestion
```

Reply:
That's why it's in draft right now. There are issues on discord's end that I've been trying to narrow down before this is ready for review. The print statements were added during troubleshooting and will be removed once this isn't a draft.